src/main/java/org/eclipse/tractusx/ssi/lib/proof/IVerifier.java

```diff
@@ -39,18 +38,16 @@ public interface IVerifier {
    * model to get the public key of issuer.
    *
    * @param hashedLinkedData the hashed linked data
-   * @param document {@link VerifiableCredential} the verifiable
+   * @param verifiable {@link VerifiableCredential} the verifiable
    * @return boolean if verified or not
    * @throws UnsupportedSignatureTypeException
```

Comment:
Seems unused.

Reply:
verifiable is a method argument of verify method